tests/e2e/tests/test_sanity.py

```diff
@@ -104,6 +108,22 @@ def on_create():
         assert retcode == 0
         lb_deps["config"] = load_yaml_file(LB_CONFIG_FILE)
 
+        #update dns record for kserve domain
+        subdomain_hosted_zone_id = lb_deps["config"]["route53"]["subDomain"]["hostedZoneId"]
+        subdomain_hosted_zone = Route53HostedZone(domain=subdomain_name, region=region, id = subdomain_hosted_zone_id)
+        wait_for_alb_dns(cluster, region)
+        ingress = get_ingress(cluster, region)
+        alb_dns = ingress["status"]["loadBalancer"]["ingress"][0]["hostname"]
+        wait_for_alb_status(alb_dns, region)
+
+        _kserve_record = subdomain_hosted_zone.generate_change_record(
```

Comment:
nit: why is this prefixed with _

Reply:
https://github.com/awslabs/kubeflow-manifests/blob/100eda733684e33fd5f781957fef14e9c35eb6b5/tests/e2e/utils/load_balancer/setup_load_balancer.py#L222